README.md

```diff
@@ -720,7 +718,8 @@ It is part of the [CSS Text Level 3](https://drafts.csswg.org/css-text-3/#hyphen
 
 ## DEPRECATED
 
-- Option `debug` will be deprecated in further versions
+- Option `debug` will be deprecated in further versions;
+- Option `html` will be deprecated in further versions.
```

Comment:
Hi. I have a question regarding this. Does this mean that in future versions I will not be able to turn off the skipping of HTML tags? I'm asking because we are using **hyphen** in a word processing application where the last word of every line is hyphenated regardless of it being HTML or not.

Reply:
Hi, I thought about making default library behavior to work with html with no need for extra configuration, that's why I changed the default value of `html` tag to be `true`

Reply:
So you say in your case html tags should be hyphenated anyway?

Reply:
I will try to come up with new option with more intuitive name and write a migration section in readme file

Reply:
> So you say in your case html tags should be hyphenated anyway?

Yes. I want to hyphenate everything I pass to the library. Although I have no problem with "html" being set to "true" by default as long as I can set it to "false".